manifest.json

```
@@ -4,266 +4,305 @@
   },
   {
     "example": "Svelte",
     "path": "/svelte-functions",
     "demo": "https://svelte-functions.now-examples.now.sh",
     "description": "A Svelte app, using the Svelte template, backed by serverless functions.",
-    "icon": "/.github/images/svelte.svg"
+    "icon": "/.github/images/svelte.svg",
+    "tagline": "Svelte lets you write high performance reactive apps with significantly less boilerplate. "
   },
   {
     "example": "Vue.js",
     "path": "/vue",
     "demo": "https://vue.now-examples.now.sh",
     "description": "A Vue.js app, created with the Vue CLI.",
-    "icon": "/.github/images/vue.svg"
+    "icon": "/.github/images/vue.svg",
+    "tagline": "Vue.js is a versatile JavaScript framework that is as approachable as it is performant."
   },
   {
     "example": "Angular",
     "path": "/angular",
     "demo": "https://angular.now-examples.now.sh",
     "description": "An Angular app, created with the Angular CLI.",
-    "icon": "/.github/images/angular.svg"
+    "icon": "/.github/images/angular.svg",
+    "tagline": "Angular is a TypeScript-based cross-platform framework from Google."
   },
   {
     "example": "Hugo",
     "path": "/hugo",
     "demo": "https://hugo.now-examples.now.sh",
     "description": "A Hugo site, created with the Hugo CLI.",
-    "icon": "/.github/images/hugo.svg"
+    "icon": "/.github/images/hugo.svg",
+    "tagline": "Hugo is the world’s fastest framework for building websites, written in Go."
   },
   {
     "example": "Sapper",
     "path": "/sapper",
     "demo": "https://sapper.now-examples.now.sh",
     "description": "A Sapper app, using the Sapper template.",
-    "icon": "/.github/images/svelte.svg"
+    "icon": "/.github/images/svelte.svg",
+    "tagline": "Sapper is a framework for building high-performance universal web apps with Svelte."
   },
   {
     "example": "UmiJS",
     "path": "/umijs",
     "demo": "https://umijs.now-examples.now.sh",
     "description": "An UmiJS app, created using the Umi CLI.",
-    "icon": "/.github/images/umi.svg"
+    "icon": "/.github/images/umi.svg",
+    "tagline": "UmiJS is an extensible enterprise-level react application framework."
   },
   {
     "example": "Vanilla",
     "path": "/vanilla-functions",
     "demo": "https://vanilla-functions.now-examples.now.sh",
     "description": "A vanilla site, backed by serverless function.",
-    "icon": "/.github/images/vanilla.svg"
+    "icon": "/.github/images/vanilla.svg",
+    "tagline": "Love the original way of doing websites?"
   },
   {
     "example": "Storybook",
     "path": "/storybook",
     "demo": "https://storybook.now-examples.now.sh",
     "description": "A Storybook app, using the React demo template.",
-    "icon": "/.github/images/storybook.svg"
+    "icon": "/.github/images/storybook.svg",
+    "tagline": "Storybook is an open source tool for developing UI components in isolation for React, Vue, and Angular."
```

```suggestion
    "tagline": "Storybook is an open source tool for developing UI components in isolation for JavaScript frameworks."
```
It covers a lot more than just the biggest frameworks.

---

I disagree with this one. 

Because we're optimising for search here on a `meta` tag, and my theory is that people are more likely to search against specific frameworks (as opposed to a generic "web framework")

e.g people are likely to search for "UI components in Vue" vs "UI components in JavaScript frameworks"